errors/errors.go

```diff
@@ -30,15 +29,28 @@ func IncorrectNumberOfArguments() {
 	os.Exit(1)
 }
 
+func InvalidArgument() {
+	color.Red("FAILED")
+	fmt.Println("You have entered an invalid argument.")
+	fmt.Println("Enter 'cf backup' to check the list of commands and their usage.")
+	os.Exit(1)
+}
+
 func IncorrectSpace(orgName string, spaceName string) {
-	color.Red("FAILED")	
-	fmt.Println("Instance name requested doesn't belong to the org: " +  orgName +  " and the space: " + spaceName + " Please target the correct org and space.")
+	color.Red("FAILED")
+	fmt.Println("Instance name requested doesn't belong to the org: " + orgName + " and the space: " + spaceName + " Please target the correct org and space.")
 	os.Exit(2)
 }
 
 func IncorrectInstanceName(instanceName string) {
 	color.Red("FAILED")
-	fmt.Println("Service Instance \"" + instanceName +  "\" doesn't exist.")
+	fmt.Println("Service Instance \"" + instanceName + "\" doesn't exist.")
+	os.Exit(3)
+}
+
+func IncorrectGuid(instanceGuid string) {
+	color.Red("FAILED")
+	fmt.Println("Service Instance Guid \"" + instanceGuid + "\" doesn't exist.")
```

Comment:
After removing [this block](https://github.com/cloudfoundry-incubator/service-fabrik-cli-plugin/pull/2/files#diff-c1200ceabb9994d8786d2389fae289c1R205) as per review comments, please check if you can delete this function as it is not used anywhere.

Reply:
Now it is used in backup.go line number - 331